e2e/tests/services.rs

```diff
@@ -226,12 +226,13 @@ impl OrderbookServices {
         let maintenance = ServiceMaintenance {
             maintainers: vec![db.clone(), event_updater],
         };
-        orderbook::serve_task(
+        orderbook::serve_api(
             db.clone(),
             orderbook,
             fee_calculator,
             price_estimator.clone(),
             API_HOST[7..].parse().expect("Couldn't parse API address"),
+            pending::<()>(),
```

Comment:
I wonder if the explicit type parameter is still needed given that `serve_api` no longer generic over the output of the future type?
```suggestion
            pending(),
```